lib/rate-limit.js

```diff
@@ -1,3 +1,5 @@
+let attempt = 0
```

Comment:
we don't want to retry forever that's why we need to have attempt there and not per request